### PR TITLE
pressure_of_ideal_gas_from_height_and_temperature

### PR DESCRIPTION
**Pull request by Darya2909**

### CODE REVIEW

File: symplyphysics/laws/thermodynamics/pressure_of_ideal_gas_from_height_and_temperature.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+final_pressure = Symbol("final_pressure", units.pressure)
+initial_pressure = Symbol("initial_pressure", units.pressure)
+atomic_weight = Symbol("atomic_weight", units.mass / units.amount_of_substance)
+final_height = Symbol("final_height", units.length)
+initial_height = Symbol("initial_height", units.length)
+temperature = Symbol("temperature", units.temperature)
+
+law = Eq(final_pressure, initial_pressure * exp(units.acceleration_due_to_gravity * atomic_weight *
+                                                (final_height - initial_height)/ (units.molar_gas_constant * temperature)))
```
Comment (blackyblack) [inline symplyphysics/laws/thermodynamics/pressure_of_ideal_gas_from_height_and_temperature.py:32]:
```suggestion
                                                (final_height - initial_height) / (units.molar_gas_constant * temperature)))
```

File: symplyphysics/laws/thermodynamics/pressure_of_ideal_gas_from_height_and_temperature.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+## The gas is in a uniform gravity field
+
+final_pressure = Symbol("final_pressure", units.pressure)
+initial_pressure = Symbol("initial_pressure", units.pressure)
+atomic_weight = Symbol("atomic_weight", units.mass / units.amount_of_substance)
+final_height = Symbol("final_height", units.length)
+initial_height = Symbol("initial_height", units.length)
+temperature = Symbol("temperature", units.temperature)
+
+law = Eq(final_pressure, initial_pressure * exp(units.acceleration_due_to_gravity * atomic_weight *
```
Comment (blackyblack) [inline symplyphysics/laws/thermodynamics/pressure_of_ideal_gas_from_height_and_temperature.py:31]: Where is minus on gravity acceleration?

File: test/thermodynamics/pressure_of_ideal_gas_from_height_and_temperature_test.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    SI,
+    convert_to
+)
+
+from symplyphysics.laws.thermodynamics import pressure_of_ideal_gas_from_height_and_temperature as pressure_of_ideal_gas
+
+
+@fixture(name="test_args")
+def test_args_fixture():
+    initial_pressure = Quantity(1e5 * units.pascal)
```
Comment (blackyblack) [inline test/thermodynamics/pressure_of_ideal_gas_from_height_and_temperature_test.py:16]: Add source of your numbers, if possible.

**Review by blackyblack — APPROVED** [review]

File: symplyphysics/laws/thermodynamics/pressure_of_ideal_gas_from_height_and_temperature.py (hunk 2 above)
Comment (Darya2909) [reply symplyphysics/laws/thermodynamics/pressure_of_ideal_gas_from_height_and_temperature.py:31]: When I put a minus in the law, it gives the wrong result, the same as if I count without a minus in the calculator, I did not understand why

Comment (blackyblack) [reply symplyphysics/laws/thermodynamics/pressure_of_ideal_gas_from_height_and_temperature.py:31]: Because you have input parameters: initial_height, final_height. But in your test you pass: final_height, initial_height.

File: test/thermodynamics/pressure_of_ideal_gas_from_height_and_temperature_test.py [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    units,
+    Quantity,
+    SI,
+    convert_to
+)
+
+from symplyphysics.laws.thermodynamics import pressure_of_ideal_gas_from_height_and_temperature as pressure_of_ideal_gas
+
+
+@fixture(name="test_args")
```
Comment (blackyblack) [inline test/thermodynamics/pressure_of_ideal_gas_from_height_and_temperature_test.py:14]: Why did you remove the link to the example?

Comment (Darya2909) [reply test/thermodynamics/pressure_of_ideal_gas_from_height_and_temperature_test.py:14]: Accidentally

**Review by blackyblack — APPROVED** [review]
